.github/workflows/release.yaml

```
@@ -4,6 +4,8 @@ on:
   push:
     tags:
       - "v*"
+    tags-ignore:
```

actionlint (1.7.7)

both "tags" and "tags-ignore" filters cannot be used for the same event "push". note: use '!' to negate patterns

(events)